packages/sdk/src/tests/contracts/Holograph.test.ts

```diff
@@ -0,0 +1,42 @@
+import {beforeAll, describe, expect, it} from 'vitest'
+
+import {Config} from '../../config/config.service'
+import {Holograph} from '../../contracts/Holograph'
+import {Providers} from '../../services/providers.service'
+import {REGEX} from '../../utils/transformers'
+
+const NETWORKS_MOCK = {
+  5: process.env.ETHEREUM_TESTNET_RPC ?? '',
+  80001: process.env.POLYGON_TESTNET_RPC ?? '',
+}
+
+describe('Contract class: Holograph', () => {
+  let config: Config
+  let providers: Providers
+
+  beforeAll(() => {
+    config = Config.getInstance(NETWORKS_MOCK)
+    providers = new Providers(config)
+  })
+
+  it('should be able to get the correct providers', () => {
+    const multiProviders = providers.providers
```

Comment:
`providers.providers` is kinda weird. might get confusing. maybe we can think about how to make it cleaner

Reply:
just improved that